soh/soh/Enhancements/randomizer/randomizerTypes.h

```diff
@@ -1041,6 +1041,20 @@ typedef enum {
     RSK_LACS_REWARD_COUNT,
     RSK_LACS_DUNGEON_COUNT,
     RSK_LACS_TOKEN_COUNT,
+    RSK_SHUFFLE_ENTRANCES,
+    RSK_SHUFFLE_DUNGEON_ENTRANCES,
+    RSK_SHUFFLE_OVERWORLD_ENTRANCES,
+    RSK_SHUFFLE_INTERIOR_ENTRANCES,
+    RSK_SHUFFLE_GROTTO_ENTRANCES,
+    RSK_SHUFFLE_OWL_DROPS,
+    RSK_SHUFFLE_WARP_SONGS,
+    RSK_SHUFFLE_OVERWORLD_SPAWNS,
+    RSK_MIXED_ENTRANCE_POOLS,
+    RSK_MIX_DUNGEON_ENTRANCES,
+    RSK_MIX_OVERWORLD_ENTRANCES,
+    RSK_MIX_INTERIOR_ENTRANCES,
+    RSK_MIX_GROTTO_ENTRANCES,
+    RSK_DECOUPLED_ENTRANCES,
```

Comment:
I threw in the rest of the entrance rando options in this enum to reserve their positions for when the v2 PR comes in later. If we don't like that, I can remove them.